Team-Guidelines.md

```diff
@@ -0,0 +1,73 @@
+# GridTools Team Guidelines
+
+## Preamble
+
+These are guidelines. There is no guideline without exception. This is a living document, nothing is written in stone.
+
+## Pull Requests
+
+### Keep PRs small
+
+Rationale:
+- Smaller PRs are easier to review, increasing code quality. Reviews on large PRs tend to focus on details as the overall design is lost in the size. The possiblity to [learn from the review](#review-is-a-platform-for-learning-and-knowledge-transfer)
+- Small PRs encourage [faster review](#Review-in-a-timely-manner).
+
+### PRs are squash merged
+
+PRs are squash merged. Take the PR description as commit message of the squashed commit. This requires a complete, concise description of the changes in the description of the PR. Add the description when the PR is opened, it is valuable input for the reviewer.
+
+Rationale:
+- We see advantage of a clean, linear history where every commit passes tests. (Don't forget to [keep PRs small](#Keep-PRs-small)!)
+
+## Code Reviews
+
+### Review in a timely manner
+
+Reviews are first priority. If you have a pending review, do it first before you start a new task.
+Use context switches to do a review instead of going back to your previous (development) task (e.g. in the morning, after a meeting, after a break). Ideally, you get a first feedback within hours.
+
```

Comment:
Could we also mention how to deal with unnecessarily large PRs? Should they be systematically rejected to force the author to open a bunch of new smaller PRs or should they reviewed and accepted anyway?

Reply:
I added a sentence to "Keep PRs small":
> Reviewer and reviewee should decide together if it makes sense to split a large PR into smaller self-contained units.

What do you think?

Reply:
Fine for me. Thanks